src/Routing/Route/Route.php

```diff
@@ -145,12 +145,12 @@ public function __construct(string $template, array $defaults = [], array $optio
             return true;
         };
 
-        assert($checker());
```

Comment:
no idea why this assert was here

Reply:
It's there so that the check only runs during development. Please add it back (and ignore whatever phpstan is complaining about).